recipes/kronprod/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "kronprod" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/Jbwasse2/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 2fefedf5d3077a88a77aeb237d7b0ef3c1e91b645ecbe92db5047a5fddd1e10c 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<34]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
```

I guess you're using the C bindings of numpy? If so you should [pin it to an ABI compatible version](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy):
```suggestion
  - {{ pin_compatible('numpy') }}
```

---

I made the recommended changes to the meta.yaml, plus I also got rid of `build:    - {{ compiler('c') }}` since I realized I dont need it. Looking at the article you linked, adding `{{ pin_compatible('numpy') }}` looks like a good idea.